Assets/Scripts/Models/Character.cs

```diff
@@ -191,29 +194,31 @@ void GetNewJob()
             DestTile = CurrTile;
         }
     }
-    
+
     void Update_DoJob(float deltaTime)
     {
-        // Do I have a job?
-        if (myJob == null)
-        {
-            GetNewJob();
-        }
-
-        if (CheckForJobMaterials()) //make sure all materials are in place
-        {
-            // If we get here, then the job has all the material that it needs.
-            // Lets make sure that our destination tile is the job site tile.
-            DestTile = JobTile;
+        if (jobWaitTimer > 0) {
+            jobWaitTimer -= 1;
+            //Logger.LogInfo (jobWaitTimer);
+        } else {
+            // Do I have a job?
+            if (myJob == null) {
+                GetNewJob ();
+            }
 
-            // Are we there yet?
-            if (CurrTile == DestTile)
-            {
-                // We are at the correct tile for our job, so 
-                // execute the job's "DoWork", which is mostly
-                // going to countdown jobTime and potentially
-                // call its "Job Complete" callback.
-                myJob.DoWork(deltaTime);
+            if (CheckForJobMaterials ()) { //make sure all materials are in place
```

Comment:
Comments are on new lines as well, have a space between `//` and the text, the text starts with a capital letter, and ends with a period.